test/e2e/lib/components/find-a-domain-component.js

```diff
@@ -103,7 +103,7 @@ export default class FindADomainComponent extends AsyncBaseContainer {
 
 	async skipSuggestion() {
 		// currently used in 'launch-site' and 'new-launch' signup flows
-		const skipSuggestion = By.css( '.domain-skip-suggestion > .button.domain-suggestion__action' );
+		const skipSuggestion = By.css( '.domain-skip-suggestion .button.domain-suggestion__action' );
```

Comment:
Is this change really needed? The `domainStepCopyUpdates` AB test doesn't seem to change the markup of the `<DomainSkipSuggestion>` component -- the `button` is still a direct descendant. It only changes some text labels, is that right?

Reply:
Oh, you're right. Double checked now and the layout is indeed the same. After following the [discussions on the re-revert PR](https://github.com/Automattic/wp-calypso/pull/45618#discussion_r488536455), I realise the problem was elsewhere.